Test suite doesn't compile when built from source tarball

Hi,

It looks like the mtbl 1.0.0 tarball doesn't include all the files needed to build the test suite:

```
edmonds@chase{0}:/tmp$ wget -nv https://dl.farsightsecurity.com/dist/mtbl/mtbl-1.0.0.tar.gz
2017-03-26 21:14:37 URL:https://dl.farsightsecurity.com/dist/mtbl/mtbl-1.0.0.tar.gz [414863/414863] -> "mtbl-1.0.0.tar.gz" [1]
edmonds@chase{0}:/tmp$ tar xf mtbl-1.0.0.tar.gz 
edmonds@chase{0}:/tmp$ cd mtbl-1.0.0
edmonds@chase{0}:/tmp/mtbl-1.0.0$ ./configure && make -j8 && make -j8 check
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for style of include used by make... GNU
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking dependency style of gcc... gcc3
checking for gcc option to accept ISO C99... none needed
checking for gcc option to accept ISO Standard C... (cached) none needed
checking how to run the C preprocessor... gcc -E
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking minix/config.h usability... no
checking minix/config.h presence... no
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... yes
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... no
checking whether make supports nested variables... (cached) yes
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking how to print strings... printf
checking for a sed that does not truncate output... /bin/sed
checking for fgrep... /bin/grep -F
checking for ld used by gcc... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
checking the name lister (/usr/bin/nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking whether the shell understands some XSI constructs... yes
checking whether the shell understands "+="... yes
checking how to convert x86_64-unknown-linux-gnu file names to x86_64-unknown-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-unknown-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for ar... ar
checking for archiver @FILE support... @
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/bin/nm -B output from gcc object... ok
checking for sysroot... no
checking for mt... mt
checking if mt is a manifest tool... no
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... no
checking for gcc option to produce PIC... -fPIC -DPIC
checking if gcc PIC flag -fPIC -DPIC works... yes
checking if gcc static flag -static works... yes
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for liblz4... yes
checking for LZ4_compress_HC in -llz4... yes
checking for libzstd... yes
checking whether byte ordering is bigendian... no
checking for mkstemp... yes
checking for posix_madvise... yes
checking for madvise... yes
checking sys/endian.h usability... no
checking sys/endian.h presence... no
checking for sys/endian.h... no
checking endian.h usability... yes
checking endian.h presence... yes
checking for endian.h... yes
checking snappy-c.h usability... yes
checking snappy-c.h presence... yes
checking for snappy-c.h... yes
checking for snappy_compress in -lsnappy... yes
checking zlib.h usability... yes
checking zlib.h presence... yes
checking for zlib.h... yes
checking for deflate in -lz... yes
checking for library containing dlopen... -ldl
checking for library containing clock_gettime... none required
checking for clock_gettime... yes
checking for a2x... /usr/bin/a2x
checking if LD -Wl,--version-script works... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating mtbl/libmtbl.pc
config.status: creating config.h
config.status: executing depfiles commands
config.status: executing libtool commands

    mtbl 1.0.0

        compiler:               gcc
        cflags:                 -g -O2
        ldflags:                
        libs:                   -ldl -lz -lsnappy 

        prefix:                 /usr/local
        sysconfdir:             ${prefix}/etc
        libdir:                 ${exec_prefix}/lib
        includedir:             ${prefix}/include
        pkgconfigdir:           ${libdir}/pkgconfig

        bigendian:              no

        building manpage docs:  yes (asciidoc available)

make  all-am
make[1]: Entering directory '/tmp/mtbl-1.0.0'
  CC       libmy/crc32c-sse42.lo
  CC       libmy/crc32c.lo
  CC       libmy/crc32c-slicing.lo
  CC       libmy/heap.lo
  CC       libmy/my_fileset.lo
  CC       src/mtbl_dump.o
  CC       src/mtbl_info.o
  CC       src/mtbl_verify.o
  CC       src/mtbl_merge.o
  CC       mtbl/block.lo
  CC       mtbl/block_builder.lo
  CC       mtbl/compression.lo
  CC       mtbl/crc32c_wrap.lo
  CC       mtbl/fileset.lo
  CC       mtbl/fixed.lo
  CC       mtbl/iter.lo
  CC       mtbl/merger.lo
  CC       mtbl/reader.lo
  CC       mtbl/sorter.lo
  CC       mtbl/source.lo
  CC       mtbl/metadata.lo
  CC       mtbl/varint.lo
  CC       mtbl/writer.lo
  CCLD     mtbl/libmtbl.la
ar: `u' modifier ignored since `D' is the default (see `U')
  CCLD     src/mtbl_dump
  CCLD     src/mtbl_info
  CCLD     src/mtbl_verify
  CCLD     src/mtbl_merge
make[1]: Leaving directory '/tmp/mtbl-1.0.0'
make  t/test-block_builder t/test-crc32c t/test-fileset-partition t/test-fixed t/test-metadata t/test-varint t/test-vector t/test-iter-seek t/test-compression
make[1]: Entering directory '/tmp/mtbl-1.0.0'
  CC       t/test-block_builder.o
  CC       t/test-crc32c.o
  CC       t/test-fileset-partition.o
  CC       t/test-fixed.o
  CC       t/test-metadata.o
  CC       t/test-varint.o
  CC       t/test-vector.o
  CC       t/test-iter-seek.o
t/test-metadata.c:11:33: fatal error: ../libmy/b64_encode.c: No such file or directory
 #include "../libmy/b64_encode.c"
                                 ^
compilation terminated.
Makefile:1071: recipe for target 't/test-metadata.o' failed
make[1]: *** [t/test-metadata.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/tmp/mtbl-1.0.0'
Makefile:1748: recipe for target 'check-am' failed
make: *** [check-am] Error 2
```

--- a/Makefile.am
+++ b/Makefile.am
@@ -130,7 +130,10 @@ t_test_fixed_LDADD = mtbl/libmtbl.la
 
 TESTS += t/test-metadata
 check_PROGRAMS += t/test-metadata
-t_test_metadata_SOURCES = t/test-metadata.c
+t_test_metadata_SOURCES = \
+	t/test-metadata.c \
+	libmy/b64_encode.c \
+	libmy/b64_encode.h
 t_test_metadata_LDADD = mtbl/libmtbl.la
 
 TESTS += t/test-varint

--- a/t/test-metadata.c
+++ b/t/test-metadata.c
@@ -7,8 +7,9 @@
 
 #include <mtbl.h>
 
+#include "libmy/b64_encode.h"
+
 #include "metadata.c"
-#include "../libmy/b64_encode.c"
 
 #define NAME	"test-metadata"
 

